docs/alerts/scheduled-searches/deprecation.md

```diff
@@ -31,6 +27,10 @@ Real-Time Scheduled Searches that were created up till **May 15, 2024** will con
 
 ## What do I need to do?
 
-Before **May 15, 2025**, please migrate any Real Time Scheduled Searches to either Monitors or reduce its frequency to the minimum frequency of 15m.
+Before **May 15, 2025**, please migrate any Real-Time Scheduled Searches to either Monitors or reduce its frequency to the minimum frequency of 15m.
+
+### Can I imported a scheduled search into a monitor?
+
+No. Because the JSON formatting of Scheduled Searches diffs from monitors, you'll need to create a monitor manually from the Search UI for your real-time use cases.
```

Comment:
```suggestion
### Can I import a scheduled search into a monitor?

No. Because the JSON formatting of Scheduled Searches differs from monitors, you'll need to create a monitor manually from the Search UI for your real-time use cases.
```